slycot/math.py

```diff
@@ -584,12 +556,9 @@ def mb03wd(job, compz, n, ilo, ihi, iloz, ihiz, H, Q, ldwork=None):
     T, Z, Wr, Wi, info = _wrapper.mb03wd(
             job, compz, n, ilo, ihi, iloz, ihiz, H, Q, ldwork)
 
-    if info < 0:
-        e = ValueError(
-                "Argument '{}' had an illegal value".format(arg_list[-info-1]))
-        e.info = info
-        raise e
-    elif info > 0:
+    raise_if_slycot_error(info, arg_list)
+
+    if info > 0:
         warnings.warn(("failed to compute all the eigenvalues {ilo} to {ihi} "
```

Comment:
can also be replaced?

Reply:
Yes, there are a lot of functions which can now be simplified. I am currently going through 
`analysis.py`.

I wanted to do the docstring cleanup (#100) in a separate PR later, but now with all the modifications in the docstrings already, which is not that easy to review it is a good thing to do much work on that now.